project-base/storefront/types/gtm/enums.ts

```diff
@@ -40,6 +40,7 @@ export enum GtmProductListNameType {
     autocomplete_search_results = 'autocomplete_search_results',
     wishlist = 'wishlist',
     sharedWishlist = 'sharedWishlist',
+    last_visited_products = 'last visited products',
```

Comment:
Do we have this enum implemented anywhere? I don't see it, if it is so, please remove this and let's add it once it needed.

Reply:
I think it is better to use is unified this way. Next developer can use it without searching if it is already added.

Reply:
It is used, I am resolving this thread